src/coreComponents/mesh/mpiCommunications/CommunicationTools.cpp

```diff
@@ -1099,4 +1108,79 @@ void CommunicationTools::synchronizeFields( FieldIdentifiers const & fieldsToBeS
   synchronizeUnpack( mesh, neighbors, icomm, onDevice );
 }
 
+
+void CommunicationTools::checkSendRecv( ObjectManagerBase const & objectManager,
```

Comment:
are you using this? it seems to be commented out everywhere it's used.

Reply:
This was is a debugging check...pretty useful until we can verify that the ghosting is rock solid again.